nest/src/dashboard.service.ts

```diff
@@ -238,15 +241,21 @@ export class DashboardService {
     return runs.filter(run => {
       const sdk = run.impl["language"] as string
       const version = run.impl["version"] as string
+      const workload = run.workload
+      const workloadVariableNames = workload["settings"]["variables"].map(((v: Record<string, any>) => v["name"]))
```

Comment:
I've realised that workload.settings is a mistake.  All the same variables are duplicated in the "vars" object - and in a much more usable form, since that's an object and settings is an array.  I'm going to get rid of workload.settings.